astropy/modeling/polynomial.py

```diff
@@ -165,15 +165,15 @@ class OrthoPolynomialBase(PolynomialBase):
         degree in x
     y_degree : int
         degree in y
-    x_domain : list or None
+    x_domain : list or None, optional
         domain of the x independent variable
-    x_window : list or None
+    x_window : list or None, optional
         range of the x independent variable
-    y_domain : list or None
+    y_domain : list or None, optional
         domain of the y independent variable
-    y_window : list or None
+    y_window : list or None, optional
         range of the y independent variable
-    param_dim : int
+    param_dim : int, optional
```

Comment:
This argument is actually deprecated.  I'd just remove the reference to it outright.  The correct argument to use is `n_models` which is mentioned in the `Model` base class, I think.  The Model doctrings still need broader reworking in order to better document the optional keyword arguments that are common to most model classes.